ci-operator/step-registry/baremetal/lab/agent/day2/baremetal-lab-agent-day2-ref.yaml

```
@@ -0,0 +1,74 @@
+ref:
+  as: baremetal-lab-agent-day2
```

```suggestion
  as: baremetal-lab-agent-day2
  cli: latest
```

See other comment about extracting oc